src/Command/Migrate.php

```diff
@@ -11,7 +11,7 @@
  */
 namespace Migrations\Command;
 
-use Cake\Event\EventManagerTrait;
+use Cake\Event\EventDispatcherTrait;
```

Comment:
I guess that this also means that you need to bump the minimal required version of cakephp to 3.0.10 ?